src/Public/css/planner.css

```diff
@@ -58,29 +58,68 @@ body {
 }
 #central-status, #res-status {
   position: relative;
-  top: calc(100% - 60px);
+  top: calc(100% - 33px);
   left: -19px;
   width: calc(100% + 38px);
   height: 32px;
   padding-top: 5px;
 }
-#res-status {
-  top: calc(100% - 33px);
-}
 .result-container {
   width: 550px;
 }
 .result-container .alert {
   width: 100%;
   padding: 5px 15px 3px 15px;
-  margin: 0;
-}
-#lastfm-attr {
-  position: relative;
-  top: calc(100% - 70px);
-  right: 1em;
-  text-align: right;
+  margin: 5px 0;
+}
+.result-container .alert label {
+  margin-top: 6px;
+}
+.result-container .alert-dismissable {
+    padding-right: 35px;
+}
+/* Spin while saving tracks */
+.gly-spin {
+  -webkit-animation: spin 2s infinite linear;
+  -moz-animation: spin 2s infinite linear;
+  -o-animation: spin 2s infinite linear;
```

Comment:
Do we still need the css prefixes? Hasn't CSS3 all been nice and standardised for ages now? Applies to the bits below this as well